x-pack/plugins/serverless_search/public/application/components/index_management/connector_empty_prompt.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButtonEmpty, EuiPanel } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { EmptyConnectorsPrompt } from '../connectors/empty_connectors_prompt';
+
+interface ConnectorIndexEmptyPromptProps {
+  indexName: string;
+  onBackClick?: () => void;
+}
+
+export const ConnectorIndexEmptyPrompt = ({ onBackClick }: ConnectorIndexEmptyPromptProps) => {
+  return (
+    <EuiPanel>
+      <EuiButtonEmpty
+        data-test-subj="serverlessSearchConnectorIndexEmptyPromptBackButton"
+        onClick={onBackClick}
+        iconSide="left"
+        iconType="arrowLeft"
+      >
+        <FormattedMessage
+          id="xpack.serverlessSearch.indexManagement.indexDetails.overview.emptyPrompt.connector.backBtn"
```

Comment:
nit: we should add this to our i18n.ts list of strings if there's not already a BACK_LABEL there (I think there is)